Trigger/src/ReadTriggerInfo_module.cc

```diff
@@ -339,9 +298,9 @@ namespace mu2e {
   void     ReadTriggerInfo::bookTrigInfoHist         (art::ServiceHandle<art::TFileService> & Tfs, summaryInfoHist_       &Hist){
     art::TFileDirectory trigInfoDir = Tfs->mkdir("trigInfo");
     Hist._hTrigBits      = trigInfoDir.make<TH1F>("hTrigInfo_bits"      , "Trigger bits"                               , 51, -0.5, 50.5);
-    for (size_t i=0; i< _trigPaths.size(); ++i){
-      Hist._hTrigBits->GetXaxis()->SetBinLabel(i+1, _trigPaths[i].c_str());
-    }
+    // for (size_t i=0; i< _trigPaths.size(); ++i){
+    //    Hist._hTrigBits->GetXaxis()->SetBinLabel(i+1, _trigPaths[i].c_str());
+    // }
```

Comment:
With a few exceptions, please no not leave commented out code in files.  You can use the git history mechanism to see removed code. One exception is if the code is currently under development but not ready to go.  If the code is for debug/testing purposes, we prefer that you keep it present but protect it with a run-time configuration varaible named something like debug_level or verbosity, with 0 meaning no output and higher numbers meaning more output.  The case for commenting out such code is if it will consume a lot of resources even if you it is never called.